Sources/Tracking/CustomerIO.swift

```diff
@@ -1,5 +1,9 @@
 import CioInternalCommon
+import FirebaseMessaging
 import Foundation
+#if canImport(UIKit)
+import UIKit
+#endif
```

Comment:
```suggestion
```

I think this was forgotten to be removed.

Reply:
✅